handlers/EveryoneAndHereMentionMessage.js

```diff
@@ -0,0 +1,24 @@
+const Discord = require('discord.js');
+
+const color = '#eb1540';
+const title = 'Nie wołaj wszystkich!';
+const thumbnail = 'LINK'
+const description = '\nRozumiemy, że potrzebujesz pomocy, ale nie wszyscy chcą zostać o tym powiadomieni.\n Jest nas tutaj dużo i nie ma sensu, aby każdy dostał bezpośrednio taką informację.\n Nie trudno sobie wyobrazić jak irytujące byłoby, gdyby każdy wołał wszystkich do każdego tematu.\n Dlatego zadaj pytanie i po prostu poczekaj - jeśli ktoś będzie wiedział i mógł, to na pewno spróbuje odpowiedzieć.'
+
+export function mentionHandler(){
+    const pingEmbed = new Discord.MessageEmbed()
+	.setColor(color)
+	.setTitle(title)
+    .setThumbnail(thumbnail)
+    .setDescription(description);
+
+    const mentionedEveryone = message.content.includes("@everyone") || message.content.includes("@here")
+    const hasPermission = message.member.hasPermission('MENTION_EVERYONE')
+    if (mentionedEveryone && !hasPermission) {
+        try{
```

Comment:
`try..catch` jest tu zbędny.

Reply:
Nie, ponieważ jak nie będzie try, catch, to bot by scrashowal jak ktos napisze @here albo @everyone w prywatnej wiad. do bota

Reply:
A co konkretnie się wtedy crashuje?

Reply:
Ok, ale to i tak trzeba obsłużyć. Zobacz jaki błąd w takiej sytuacji wyskakuje, wtedy możnabygo ignorować. Ale inne błędy powinny zostać co najmniej zlogowane albo optymalnie wysłane do jakiegos prawilnego loggera albo sentry.

Reply:
Sprawdziłem i błąd można zignorować, ponieważ dotyczy on tego, że nie może sprawdzić w prywatnej wiadomości czy ktoś ma permisję. Bez try..catch bot wprost się wyłączy.

Reply:
> nie może sprawdzić w prywatnej wiadomości czy ktoś ma permisję
Czy API Discorda nie pozwala na sprawdzenie czy wiadomość jest prywatna? Można by wtedy zapisać warunek, że bot obsłuży tylko wiadomość publiczną.

Na SO coś o tym piszą i w docsach też coś na ten temat jest:
https://stackoverflow.com/a/48729333/4983840
https://discord.com/developers/docs/resources/channel#channel-object-channel-types

Reply:
Dobrze, ale to jeden case. Tak jak pisałem wyżej, mogą się pojawiać też inne błędy które tutaj złapiesz ale ich nie obsłużysz. Np. co się stanie jeżeli bot nie będzie mógł wysłać wiadomości na serwer z powodu utraty połączenia/zmiany secretu? Na podstawie tego catcha taka sytuacja będzie nie do odróżnienia od przypadku który opisujesz.